app/helpers/rails_performance/rails_performance_helper.rb

```diff
@@ -132,6 +132,8 @@ def active?(section)
         "is-active" if controller_name == "rails_performance" && action_name == "crashes"
       when :requests
         "is-active" if controller_name == "rails_performance" && action_name == "requests"
+      when :resources
+        "is-active" if controller_name == "rails_performance" && action_name == "resources"
```

Comment:
I think code would be more clear if we check `controller_name == "rails_performance"` at the beginning of method by guard

```ruby
return false unless controller_name == "rails_performance"
```

without checking this each time

Reply:
you can change it your PR, I don't mind, I actually didn't notice it :) just copy-paste :)